.gitignore

```diff
@@ -97,7 +97,7 @@ fabric.properties
 # Module directory
 .terraform/
 
-.env
+**/.env
```

Comment:
All of the removal in other .gitignore files are related to consolidation done here on the top level

Reply:
🧹 nice one!